src/main/java/gov/cms/madie/models/measure/CodeSystemVersion.java

```diff
@@ -0,0 +1,14 @@
+package gov.cms.madie.models.measure;
+
+
+import lombok.Data;
+
+@Data
+public class CodeSystemVersion {
```

Comment:
can we not rename it to CodeSystem instead of CodeSystemVersion because wording of CodeSystemVersion is more confining it to  version but not to entire Code System props